queries/python/highlights.scm

```diff
@@ -237,6 +237,17 @@
 
 ["for" "while" "break" "continue"] @repeat
 
+[
+  "try"
+  "except"
+  "raise"
+  "finally"
+] @exception
```

Comment:
can you remove them  from `@keyword`?

Reply:
[4f8d071](https://github.com/nvim-treesitter/nvim-treesitter/pull/2738/commits/4f8d07182a982e06fcef900cf8e1e78412c8507a) sure